Dockerfile-runnable

```diff
@@ -0,0 +1,2 @@
+FROM datagovsg/airflow-pipeline:latest
+MAINTAINER Chris Sng <chris@data.gov.sg>
```

Comment:
Apparently `MAINTAINER` is deprecated: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Should use more flexible `LABEL` in future.

Reply:
Ok noted.